src/xlsx/extraction/cf_extractor.ts

```diff
@@ -124,6 +120,19 @@ export class XlsxCfExtractor extends XlsxBaseExtractor {
     };
   }
 
+  private extractCfDataBar(
+    cfRulesElement: Element,
+    theme: XLSXTheme | undefined
+  ): XLSXDataBar | undefined {
+    const dataBarElement = this.querySelector(cfRulesElement, "dataBar");
+    if (!dataBarElement) return undefined;
+    return {
+      color: this.extractColor(dataBarElement.querySelector("color"), theme, "ffffff"),
```

Comment:
White doesn't seems like a good default color for a databar, it will be invisible 😛